NEWS

```diff
@@ -13,8 +13,14 @@
 
 ## 1.16.0-rc4 (February 21, 2024)
 ### Bugfixes:
+#### UCP
+* Disabled rendezvous pipeline protocol selection when using non-contiguous buffer.
 #### UCT
 * Fixed Infiniband posting error when compiler optimizes code with memmove()
+#### GPU (CUDA, ROCM)
+* Fixed CUDA IPC disabling when CUDA device is not set for the process.
```

Comment:
Fixed cuda_ipc transport being disabled if a CUDA device is not set during initialization